server/src/game/messages.rs

```diff
@@ -0,0 +1,40 @@
+use actix::prelude::*;
+
+use super::server_actor::UserId;
+
+/// Define message types between server and clients
```

Comment:
Module-level documentation should appear at the top of the file and begin with `//!`.